it/uk/gov/hmrc/agentsubscriptionfrontend/controllers/SignOutControllerISpec.scala

```diff
@@ -64,7 +64,7 @@ class SignOutControllerISpec extends BaseISpec {
       implicit val request: FakeRequest[AnyContentAsEmpty.type] = authenticatedAs(subscribingAgentEnrolledForNonMTD)
       sessionStoreService.currentSession.continueUrl = Some(ourContinueUrl)
 
-      val result = await(controller.redirectToSos(request))
+      val result = await(controller.redirectTaskListUserToCreateCleanCreds(request))
 
       val sosContinueValueUnencoded =
```

Comment:
Any tests for the redirectUserToCreateCleanCreds method?

Reply:
Making some now...